program/unrar/ck-compile.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/sh
```

Comment:
Now days there is another modern way to reduce boilerplate and C&P scripts 
It's not critical for this issue but just please take look at 
/home/daniil/CK/ck-autotuning/package/lib-protobuf-3.1.0-android/.cm/meta.json 
you can add reference at preprocess script 
```
  "use_preprocess_scripts_from_another_entry": {
    "module_uoa": "84e27ad9dd12e734",
    "data_uoa": "fd44428fbc77f77f"
  },
  "use_scripts_from_another_entry": {
    "data_uoa": "cd9ccc74060b3d18",
    "module_uoa": "script"
  }
```
and it will be crossplatform compatible out of the box 
but your current version work only for unix/linux like system.